applications/nrf_desktop/src/events/ble_event.c

```diff
@@ -23,18 +23,20 @@ static int log_ble_qos_event(const struct event_header *eh,
 {
 	const struct ble_qos_event *event = cast_ble_qos_event(eh);
 	int pos = 0;
+	char log_buf[BLE_EVENT_LOG_BUF_LEN];
 	int err;
 
-	err = snprintf(&buf[pos], buf_len - pos, "chmap:");
+	err = snprintf(&log_buf[pos], BLE_EVENT_LOG_BUF_LEN - pos, "chmap:");
 	for (size_t i = 0; (i < CHMAP_BLE_BITMASK_SIZE) && (err > 0); i++) {
 		pos += err;
-		err = snprintf(&buf[pos], buf_len - pos, " 0x%02x", event->chmap[i]);
+		err = snprintf(&log_buf[pos], BLE_EVENT_LOG_BUF_LEN - pos, " 0x%02x", event->chmap[i]);
 	}
 	if (err < 0) {
 		pos = err;
 	}
 
-	return pos;
```

Comment:
Please introduce error handling. Maybe explicitly inform that event's data logging function failed.

Reply:
That would make error clearly visible to user.